Subject: Cleanup: GDB does not need syment

The syment struct is not used for anything by the embedded GDB,
so we can remove it completely.

Signed-off-by: Petr Tesarik <ptesarik@suse.cz>

---
 defs.h        |    5 ++---
 gdb-7.0.patch |    4 +---
 2 files changed, 3 insertions(+), 6 deletions(-)

--- a/defs.h
+++ b/defs.h
@@ -1963,9 +1963,6 @@ struct alias_data {                 /* c
 		retaddr[i] = 0;						\
 }
 
-#endif /* !GDB_COMMON */
-
-
 #define SYMBOL_NAME_USED (0x1)
 #define MODULE_SYMBOL    (0x2)
 #define IS_MODULE_SYMBOL(SYM)  ((SYM)->flags & MODULE_SYMBOL)
@@ -1981,6 +1978,8 @@ struct syment {
 	unsigned char pad2;
 };
 
+#endif /* !GDB_COMMON */
+
 #define NAMESPACE_INIT     (1)
 #define NAMESPACE_REUSE    (2)
 #define NAMESPACE_FREE     (3)
